Etabs_Adapter/Etabs_Adapter.csproj

```diff
@@ -41,7 +41,7 @@
     <OutputType>Library</OutputType>
     <AppDesignerFolder>Properties</AppDesignerFolder>
     <RootNamespace>BH.Adapter.ETABS</RootNamespace>
-    <AssemblyName>ETABS17_Adapter</AssemblyName>
+    <AssemblyName>ETABS_Adapter</AssemblyName>
```

Comment:
The `AssemblyName` needs to stay as `ETABS17_Adapter` as this is targeting the specific version of ETABS. If you want to target another version we might need another config? @IsakNaslundBh adding you into this

Reply:
@al-fisher , yes, important that make sure the dll's for the different builds for the adapter to have different name to allow for multiple versions to be supported simultaneously, as if we keep the names the same they willoverwrite previous version of the dll when next build is ran. 

Does this make sense, and are you ok with this @RichardWhitfieldTTW

Reply:
Sorry for the very slow reply...I thought that this would requiremoreinput from me!
OK with me.

Reply:
Thanks @RichardWhitfieldTTW - no problem!